openmp/runtime/src/CMakeLists.txt

```diff
@@ -10,12 +10,19 @@
 
 include(ExtendPath)
 
+# The generated headers will be placed in clang's resource directory if present.
+if(${OPENMP_STANDALONE_BUILD})
+  set(LIBOMP_HEADERS_INTDIR ${CMAKE_CURRENT_BINARY_DIR})
+else()
+  set(LIBOMP_HEADERS_INTDIR ${LLVM_BINARY_DIR}/${LIBOMP_HEADERS_INSTALL_PATH})
```

Comment:
Is `LLVM_BINARY_DIR` still valid if user only does `-DLLVM_ENABLE_PROJECTS="openmp"`? In this case LLVM will not be probably built.

Reply:
Is it `INSTDIR` or `INTDIR`? What does `INTDIR` mean?

Reply:
Done to match `LLVM_RUNTIME_OUTPUT_INTDIR` conceptually, It's just intermediate directory. I.e. not where it's being installed, but where it's stored in the build.

Reply:
oh "intermediate"